migrations/versions/37ac005a48c3_add_draft_services.py

```diff
@@ -0,0 +1,46 @@
+"""empty message
+
+Revision ID: 37ac005a48c3
+Revises: 20_adding_json_index_to_services
+Create Date: 2015-06-02 18:32:08.399213
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '37ac005a48c3_add_draft_services'
```

Comment:
We were trying to make the revision identifiers nice and sequential and human-readable...